srcs/fs/file.rs

```diff
@@ -0,0 +1,70 @@
+use crate::boxed::Box;
+use crate::spin::Mutex;
+use crate::string::String;
+use alloc::sync::Arc;
+
+pub trait FileOperation {
+	fn read(&self, dst: &mut [u8], length: usize) -> Result<usize, FileError>;
+	fn write(&mut self, src: &[u8], length: usize) -> Result<usize, FileError>;
+}
+
+#[derive(Debug)]
+pub enum FileError {
+	Unknown()
+}
+
+/// Contains all file information.
+/// Current informatin are only it's name and it's FileOperation trait object
```

Comment:
Typo and rephrase: "Current information are only name and FileOperation trait object".. no need "it's"